src/components/Conversation.svelte

```diff
@@ -1,15 +1,14 @@
 <script>
 import { formatDistanceToNow } from 'date-fns'
-import { sendMessage, sendCommit, acceptCommittment } from '../data/gqlQueries'
+import { sendCommit, acceptCommittment } from '../data/gqlQueries'
 import { createEventDispatcher } from 'svelte'
-import { unreads, saw } from '../data/messaging'
+import { unreads, saw, send } from '../data/messaging'
 
 export let me = {}
 export let conversation = {}
 export let minimal = false
 
 let reply = ''
-const dispatch = createEventDispatcher()
```

Comment:
Review whether this needs to be instantiated here, rather than waiting until later